lib/dpl/provider/lambda.rb

```diff
@@ -197,7 +228,19 @@ def default_kms_key_arn
       def function_tags
         options[:function_tags] ? split_string_array_to_hash(options[:function_tags]) : nil
       end
-
+      
+      def batch_size
+        options[:batch_size] ? options[:batch_size] : 1
+      end
+      
+      def starting_position
+        options[:starting_position] ? options[:starting_position] : nil
```

Comment:
This ternary operator seems superfluous. Also, should we not validate the user input here?